website/docs/r/project_api_key.html.markdown

```diff
@@ -46,7 +46,7 @@ resource "mongodbatlas_project_api_key" "test" {
 ## Argument Reference
 
 * `description` - (Required) Description of this Project API key.
-* `project_id` - Unique 24-hexadecimal digit string that identifies your project.
+* `project_id` - Unique 24-hexadecimal digit string that identifies your project. **WARNING:** this parameter is deprecated as it no longer needs to be defined. It will be removed in version 1.16.0.
```

Comment:
Isn't this a breaking change? 🤔

Reply:
if the customers have to update their configuration to not use this field -> it is a breaking change

Reply:
> if the customers have to update their configuration
Customers will not have to make any changes to there existing configuration with this change, the `project_id` is made optional and deprecated as it will be eventually deleted (this is when it will be a breaking change). For reference, HashiCorp versioning guidelines allow deprecating attributes in a minor version as existing functionality remains compatible: https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#example-minor-number-increments